frontend/test/metabase-smoketest/user.cy.spec.js

```diff
@@ -27,24 +25,24 @@ describe("smoketest > new_user", () => {
     cy.findByText("Pick a column to group by").click();
     cy.findByText("Title").click();
 
-    cy.contains("Average of Rating");
+    cy.findByText("Average of Rating");
 
     cy.findByText("Visualize").click();
 
-    cy.get(".Icon-bar", { timeout: 30000 });
-    cy.contains("Vendor is not empty");
-    cy.contains("Visualization");
-
-    // =================
-    // should filter via both the header and notebook editor
-    // =================
+    cy.get(".Icon-bar");
+    cy.findAllByText("Vendor is not empty");
+    cy.findByText("Visualization");
```

Comment:
Let's find a way to ensure the visualization has loaded. In my Cypress run, these validations were completed on the notebook editor page, not the data viz page. Was that the intention?

Reply:
These validations should be completed on the data viz page; this information shouldn't have loaded until the data viz page is viewable. 

Also, I've added in a `cy.get("svg")` to check that the visualization has loaded.